{
 "PRed": [
  {
   "PR": {
    "number": 3,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "hdf51140"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "labels": [],
    "number": 5,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "hdf51141"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "labels": [],
    "number": 6,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "hdf51142"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  }
 ],
 "bad": false,
 "pinning_version": "2023.08.22.21.41.48",
 "pre_pr_migrator_attempts": {
  "boost_cpp_to_libboost": 162,
  "python311": 487
 },
 "pre_pr_migrator_status": {
  "boost_cpp_to_libboost": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/7008804982'>bot CI job</a>) @ main\n                <details>\n                <div align=\"left\">\n                <pre>\n                linux_64_python3.10.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 cgal 5.5.2.*  is installable with the potential options\n\u2502  \u251C\u2500 cgal 5.5.2 would require\n\u2502  \u2502  \u2514\u2500 boost-cpp >=1.78.0,<1.78.1.0a0 , which can be installed;\n\u2502  \u251C\u2500 cgal 5.5.2 would require\n\u2502  \u2502  \u2514\u2500 python_abi 3.11.* *_cp311, which can be installed;\n\u2502  \u251C\u2500 cgal 5.5.2 would require\n\u2502  \u2502  \u2514\u2500 python_abi 3.8.* *_cp38, which can be installed;\n\u2502  \u2514\u2500 cgal 5.5.2 would require\n\u2502     \u2514\u2500 python_abi 3.9.* *_cp39, which can be installed;\n\u251C\u2500 libboost-headers is not installable because there are no viable options\n\u2502  \u251C\u2500 libboost-headers 1.82.0 would require\n\u2502  \u2502  \u2514\u2500 boost-cpp 1.82.0* , which conflicts with any installable versions previously reported;\n\u2502  \u2514\u2500 libboost-headers 1.83.0 would require\n\u2502     \u2514\u2500 boost-cpp 1.83.0* , which conflicts with any installable versions previously reported;\n\u251C\u2500 libboost-python-devel 1.78.0.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 python 3.10.* *_cpython is not installable because it requires\n   \u2514\u2500 python_abi 3.10.* *_cp310, which conflicts with any installable versions previously reported.</pre><pre>linux_64_python3.11.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 cgal-cpp 5.5.2.*  is installable and it requires\n\u2502  \u2514\u2500 boost-cpp >=1.78.0,<1.78.1.0a0 , which can be installed;\n\u251C\u2500 libboost-headers is not installable because there are no viable options\n\u2502  \u251C\u2500 libboost-headers 1.82.0 would require\n\u2502  \u2502  \u2514\u2500 boost-cpp 1.82.0* , which conflicts with any installable versions previously reported;\n\u2502  \u2514\u2500 libboost-headers 1.83.0 would require\n\u2502     \u2514\u2500 boost-cpp 1.83.0* , which conflicts with any installable versions previously reported;\n\u2514\u2500 libboost-python-devel 1.78.0.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_64_python3.8.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 cgal-cpp 5.5.2.*  is installable and it requires\n\u2502  \u2514\u2500 boost-cpp >=1.78.0,<1.78.1.0a0 , which can be installed;\n\u251C\u2500 libboost-headers is not installable because there are no viable options\n\u2502  \u251C\u2500 libboost-headers 1.82.0 would require\n\u2502  \u2502  \u2514\u2500 boost-cpp 1.82.0* , which conflicts with any installable versions previously reported;\n\u2502  \u2514\u2500 libboost-headers 1.83.0 would require\n\u2502     \u2514\u2500 boost-cpp 1.83.0* , which conflicts with any installable versions previously reported;\n\u2514\u2500 libboost-python-devel 1.78.0.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_64_python3.9.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 cgal-cpp 5.5.2.*  is installable and it requires\n\u2502  \u2514\u2500 boost-cpp >=1.78.0,<1.78.1.0a0 , which can be installed;\n\u251C\u2500 libboost-headers is not installable because there are no viable options\n\u2502  \u251C\u2500 libboost-headers 1.82.0 would require\n\u2502  \u2502  \u2514\u2500 boost-cpp 1.82.0* , which conflicts with any installable versions previously reported;\n\u2502  \u2514\u2500 libboost-headers 1.83.0 would require\n\u2502     \u2514\u2500 boost-cpp 1.83.0* , which conflicts with any installable versions previously reported;\n\u2514\u2500 libboost-python-devel 1.78.0.*  does not exist (perhaps a typo or a missing channel).\n                </pre>\n                </div>\n                </details>",
  "python311": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/6287856858\">bot CI job</a>): main: ['linux_64_python3.11.____cpython: Encountered problems while solving:\\n  - package ifes-apt-tc-data-modeling-0.0.9-pyhd8ed1ab_1 requires python >=3.8,<3.11, but none of the providers can be installed\\n  - nothing provides hdf5 1.8.15* needed by h5py-2.6.0-np110py27_3\\n']"
 },
 "smithy_version": "3.24.1"
}